lib/blockscore/base.rb

```diff
@@ -60,8 +60,16 @@ def persisted?
       !id.nil? && !deleted?
     end
 
+    def deleted?
```

Comment:
I don't see any specs for this method. Are you running mutant against the changes you've been making? I would think it would automatically flag this for you.